src/adapters/prebidServer.js

```diff
@@ -0,0 +1,121 @@
+import Adapter from 'src/adapters/adapter';
+import bidfactory from 'src/bidfactory';
+import bidmanager from 'src/bidmanager';
+import * as utils from 'src/utils';
+import { ajax } from 'src/ajax';
+import { STATUS } from 'src/constants';
+import { queueSync, persist } from 'src/cookie.js';
+
+const TYPE = 's2s';
+const cookiePersistMessage = `Your browser may be blocking 3rd party cookies. By clicking on this page you allow Prebid Server and other advertising partners to place cookies to help us advertise. You can opt out of their cookies <a href="https://www.appnexus.com/en/company/platform-privacy-policy#choices" target="_blank">here</a>.`;
```

Comment:
It's part of using prebid server for now. So opt out would be don't use prebid server.